### PR TITLE
[3.1.0] Add API Category based search details

### PR DESCRIPTION
**Pull request by Sarangan0219**
## Purpose

Fixes https://github.com/wso2/docs-apim/issues/2282

## Affected Versions

APIM 3.1.0

### CODE REVIEW

File: en/docs/learn/consume-api/discover-apis/search.md [hunk 1]
```diff
@@ -74,6 +74,11 @@ You can search for APIs in the API Publisher or Developer Portal in the followin
 <p>A property can be defined to an API at the time it is created or later. There can be APIs without properties as this parameter is optional. <strong><br />
 </strong></p></td>
 </tr>
+<tr class="even">
+<td>By API category</td>
+<td><p><strong>api-category:xxxx</strong> . For example, api-category:test</p>
+<p>A category is used to group APIs based on any properties. API categories are defined in the Admin Portal and assigned in Design Configurations of the Publisher.</p></td>
```
Comment (wasuradananjith) [inline en/docs/learn/consume-api/discover-apis/search.md:80]: Shall we change this to the below sentence?
"An API category can be used to group APIs based on any property. API categories can be defined from the Admin Portal and can be assigned to an API in the Design Configurations section of the Publisher Portal."

Comment (Sarangan0219) [reply en/docs/learn/consume-api/discover-apis/search.md]: Makes sense :+1: 

Corrected https://github.com/wso2/docs-apim/pull/2615/commits/07830c88291acc22d63a264a647700db8ead2290

**Review by wasuradananjith — APPROVED** [review]

File: en/docs/learn/consume-api/discover-apis/search.md [hunk 2]
```diff
@@ -74,6 +74,11 @@ You can search for APIs in the API Publisher or Developer Portal in the followin
 <p>A property can be defined to an API at the time it is created or later. There can be APIs without properties as this parameter is optional. <strong><br />
 </strong></p></td>
 </tr>
+<tr class="even">
+<td>By API category</td>
+<td><p><strong>api-category:xxxx</strong> . For example, api-category:test</p>
+<p>An API category can be used to group APIs based on any property. API categories can be defined from the Admin Portal and can be assigned to an API from the Design Configurations section of the Publisher Portal.</p></td>
```
Comment (tgtshanika) [inline en/docs/learn/consume-api/discover-apis/search.md:80]: "Any property" implies that API properties are bound with API categories. Shall we reword this? "An API category can be used to group APIs"

Comment (Sarangan0219) [reply en/docs/learn/consume-api/discover-apis/search.md]: According to our offline conversation rephrased property to criteria https://github.com/wso2/docs-apim/pull/2615/commits/fe3422d92d36fc2dc410304041a563ff57dcfd06

**Review by tgtshanika — APPROVED** [review]

**Pull request merged**